action.yml

```diff
@@ -17,6 +17,8 @@ inputs:
     required: true
   tag:
     required: true
+  ssh_key_ids:
```

Comment:
If we're making this required, we should also update the typed in a few places where it's down as `sshKeyIds?: string[]` → `sshKeyIds: string[]`